engines/made/made.h

```diff
@@ -93,6 +106,8 @@ class MadeEngine : public ::Engine {
 	uint32 _cdTimeStart;
 	bool _introMusicDigital;
 
+	MADEAction _action;
```

Comment:
We don't need to hold on to the current action as a member variable as it is handled directly in the event loop.